src/ActionModal/utils/ActionManager.js

```diff
@@ -192,7 +193,7 @@ export default class ActionManager {
       txMessages.push(txMessage)
     }
 
-    const createSignedTransaction = await getTransactionSigner(context)
+    const createSignedTransaction = await signedTransactionCreator(context)
```

Comment:
I think the naming of this function can be improved. Why not `getSignedByTransactionCreator` or `signByTransactionCreator`? I think both are more descriptive names

Reply:
Why are those `signedBy`?